torch/_torch_docs.py

```diff
@@ -5298,6 +5298,40 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.heaviside,
+           r"""
+heaviside(input, val, *, out=None) -> Tensor
```

Comment:
Maybe `input, other, *, out=None`? There's not a great analogy for this naming scheme, so I was looking at how we name arguments to `torch.add`. Is there a better function to look at?

Reply:
What about `input, values, *, out=None` ? I found this from `torch.searchsorted()`.
I'm not sure does it's appropriate. If there is a more suitable name, I could replace it at any time.